src/main/java/com/nerodesk/Launch.java

```diff
@@ -73,6 +73,20 @@ public static void main(final String... args) throws Exception {
     }
 
     /**
+     * Return passed port or run on default one 8080.
+     * @param args Command line args
+     * @return Application port.
+     */
+    private static int port(final String... args) {
```

Comment:
@ggajos This is a "holy war" question :)
IMHO, 7 lines is too small. Extracting code from such a small method to the new method creates classic "spaghetti code".

Also, I know Yegor's preferences :)